### ISSUE TITLE
catchError false positive

### ISSUE THREAD

**Issue by cartant** [cartant/rxjs-tslint-rules#92]
In code like this, the placement of `catchError` outside of the map is not unsafe:

```ts
actions.pipe(
  ofType(foo),
  concatMap(...),
  catchError((error, source) => source.pipe(startWith(...)))
);
```

**Comment by cartant** [comment]
Closed by https://github.com/cartant/rxjs-tslint-rules/commit/86211bb652e146d211accaa2121a90437e499308.
